odoo/tests/common.py

```diff
@@ -141,6 +141,14 @@ class BaseCase(TreeCase, MetaCase('DummyCase', (object,), {})):
     longMessage = True      # more verbose error message by default: https://www.odoo.com/r/Vmh
     warm = True             # False during warm-up phase (see :func:`warmup`)
 
+    def addons_installed(self, *addons):
+        """Know if the specified addons are installed."""
+        found = self.env["ir.module.module"].search([
+            ("name", "in", addons),
+            ("state", "not in", ["uninstalled", "uninstallable"]),
+        ])
+        return set(addons) - set(found.mapped("name"))
```

Comment:
I don't understand why you substract it here.

If we want for example depend on module "web" the we would find "web" with the search and then we remove it at the end from the list/set and we would get an empty list:

```python
>>> set(['web']) - set(['web'])
set()
```